### DISCUSSION TITLE
[Question] WebSocketBehavior#close event handler's "No more socket updates required." message?

### DISCUSSION THREAD

**Discussion by michaelnwani** [uNetworking/uWebSockets.js#242] [Q&A] [answered]
Hi Alex,

In my `WebSocketBehavior#close` event handler, I'm sometimes seeing this value being printed from the message parameter: _No more socket updates required._

It appears to be harmless since it's accompanied with a websocket exit status code of 1000 (normal close), but after doing some google and github searching, I couldn't figure out what's causing this message to be sent. I'm on v16.3.0

Do you happen to have any ideas/suggestions regarding this? Thank you!

**Comment by ghost** [comment, marked as answer]
That message comes from a client. It's not coming from uWS. So it's just a regular close message some client is sending you, as spec allows. I would ignore it if you don't care about it.
